src/structures.rs

```diff
@@ -23,7 +23,27 @@ pub struct SvPackageDeclaration {
 #[derive(Debug, Serialize, Clone)]
 pub struct SvParameter {
     pub identifier: String,
-    pub datatype: String,
+    pub value: Option<String>,
```

Comment:
It might make sense to rename `value` to `expression`. Then when expression resolving is functional you can use `value` for the actual resolved value.

Reply:
Fixed.